srcpkgs/atom-bin/files/LICENSE

```diff
@@ -0,0 +1,20 @@
+Copyright (c) 2011-2018 GitHub Inc.
```

Comment:
No. We must NOT track licenses in our repo.

Reply:
And what if the archive doesn't contain one?